packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -535,6 +541,10 @@ export class Repository extends Construct implements IRepository {
       });
     }
 
+    if (props.securityGroups) {
```

Comment:
Then, here, similar to the RenderQueue. Replace this with:

1. Passing the filesystem SG as a [property](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-efs.FileSystem.html#securitygroupspan-classapi-icon-api-icon-experimental-titlethis-api-element-is-experimental-it-may-change-without-noticespan) when creating the EFS.
2. Passing the database SG as a [property](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-docdb.InstanceProps.html#securitygroupspan-classapi-icon-api-icon-experimental-titlethis-api-element-is-experimental-it-may-change-without-noticespan) when creating the DocDB.
3. Passing the installer's SG as a property when creating the ASG.